File: service/rate.js
class Rate {

    from;
    to;

    fromTo;
    toFrom;

    constructor(from, to, rft, rtf) {
        this.from = from;
        this.to = to;
        this.fromTo = rft;
        this.toFrom = rtf;
    }

    getFrom() {
        return this.from;
    }

    getTo() {
        return this.to;
    }

    keyToFrom() {
        return `${this.to}-${this.from}`;
    }

    rateToFrom() {
        return this.toFrom;
    }

    keyFromTo() {
        return `${this.from}-${this.to}`;
    }

    rateFromTo() {
        return this.fromTo;
    }

}

module.exports = Rate;

File: ws/src/main/java/com/soap/cotizaciones/ws/Rate.java
package com.soap.cotizaciones.ws;

public class Rate {

    private String from;
    private String to;
    private double fromTo;
    private double toFrom;

    public Rate(String from, String to, double rft, double rtf) {
        this.from = from;
        this.to = to;
        this.fromTo = rft;
        this.toFrom = rtf;
    }

    public String getFrom() {
        return from;
    }

    public String getTo() {
        return to;
    }

    public String keyToFrom() {
        return to + "-" + from;
    }

    public double rateToFrom() {
        return toFrom;
    }

    public String keyFromTo() {
        return from + "-" + to;
    }

    public double rateFromTo() {
        return fromTo;
    }
}

